Cards material

pedro-dietrich opened the pull request:
Develops issue #23.

The new GLTF card model is now being used.

The card mesh material is overridden with a custom shader, that takes the default card texture and three textures for the background, middleground and foreground of the card image.

While only Major Arcanas use three layers, it is possible to just use the background image and have the other two layers as transparent textures.

At the moment, placeholder images are being used for the three layers. They must be replaced by the actual textures.

yusei234 reviewed (changes requested):
can't validate this PR, right know one of the layer of the cards fly at the top of screen when the card goes out of the deck and disapear + the cards don't have any texture anymore.
I think this PR will be merge when there will be no more placeholder

pedro-dietrich commented:
@vdlwang , the golden ring being rendered on top is a problem that was there before I started developing on this branch. This has no relation to this PR and should be fixed in another (probably just make it the first thing to render after the background).

And in the printscreen, why there is no background image? Is another bug or something you messed with? The background texture was duplicated, so I deleted one of them. I thought I fixed all the references, because I don't get this bug on my PC.

pedro-dietrich commented:
@yusei234 I'm not being able to reproduce this bug; can you try to fix it in this PR?
Check if Godot outputs some error about not being able to find the image at a certain path; this might show where the problem is.

The actual textures should be used in another PR, at this point, we cannot afford to hold the development because of that.

pedro-dietrich commented:
I don't know if I will have much time to develop today, so feel free to push changes to this branch.
